oms.sh

```diff
@@ -22,7 +22,7 @@ service_string=$(printenv ENABLED_SERVICES)
 ## Split services into array
 services=(${service_string//:/ })
 
-command="docker-compose -f base-docker-compose.yml"
+command="sudo docker-compose -f base-docker-compose.yml"
```

Comment:
Just for information, you can run Docker for non-sudo users: https://askubuntu.com/questions/477551/how-can-i-use-docker-without-sudo
Not sure what be more secure though.

Reply:
thanks! I will think about it in the optics of continuous deployment, because right now I am just thinking development